Document/0x05b-Basic-Security_Testing.md

```diff
@@ -105,9 +105,30 @@ An extensive list of root detection methods is presented in the "Testing Anti-Re
 
 For a typical mobile app security build, you'll usually want to test a debug build with root detection disabled. If such a build is not available for testing, you can disable root detection in a variety of ways that will be introduced later in this book.
 
-#### Recommended Tools
-
--- ToDo: <https://github.com/OWASP/owasp-mstg/issues/1227>
+#### Recommended Tools - Android device
+
+There are many tools and frameworks for security testing Android applications used throughout this guide. In the next sections you will learn more about some of their main commands and interesting use cases. Please check the official documentation for installation instructions of the following tools/apks:
+
+- APK Extractor: App to extract apks without root.
+- frida server: Server for Frida, the dynamic instrumentation toolkit for developers, reverse-engineers, and security researchers.
+- objection apk: APK for objection, the runtime mobile exploration toolkit, powered by Frida. Useful with non-rooted devices.
+- drozer agent: Agent for drozer, the framework that allows you to search for security vulnerabilities in apps and devices.
+
+#### Recommended Tools - Host computer
+
+In order to analyze Android apps, you should install the following tools on your host computer. Please check the official documentation for installation instructions of the following tools/frameworks. We'll be referring to them throughout the guide.
+
+- adb: Used for communication with Android devices and debugging applications.
+- apktool: Used to pack/unpack an app executable.
+- apkx: Used retrieving an APK's source through several decompilers.
+- Burp Suite: Tnterception proxy that can be used to analyze the traffic between the app and the API it's talking to. 
+- frida: Dynamic instrumentation toolkit for developers, reverse-engineers, and security researchers.
+- mobsf: An automated, all-in-one mobile application pentesting framework.
+- objection: Runtime mobile exploration toolkit, powered by Frida. Useful with non-rooted devices.
+- radare2: Reverse engineering framework for analysis, disassembling, debugging and patching.
+- r2frida: Merges the powerful reverse engineering capabilities of radare2 with the dynamic instrumentation toolkit of Frida.
+- drozer: Framework that allows you to search for security vulnerabilities in apps and devices.
+- xposed: Framework that can change the behavior of the system and apps without touching any APKs.
```

Comment:
xposed -> Xposed 

xposed should be listed in the android device list instead

Reply:
Please align the structure to the iOS section 
https://github.com/OWASP/owasp-mstg/blob/master/Document/0x06b-Basic-Security-Testing.md#recommended-tools---host-computer

Subheadings, description and inline link

Reply:
If we align the subheading and description to the iOS section, wouldn't we have repetitive content as the description, installation instructions and commands would like on the same level (#5)  when described right above this? 

If we do this, it wouldn't be a list. We would have the same information as below when we introduce each tool right?

Reply:
Yes, you are right, but whatever is applicable for Android (Burp, Frida, Ghidra etc.) I would copy/paste from iOS for the host computer and Android device. Yes, the content would be redundant but is also applicable and important for the Android test setup. The alternative would be a list of generic tools that are applicable for iOS and Andorid in a separate chapter which would make it more fragmented. 
@commjoen Do you have a better idea?

Reply:
Talked to Carlos and he went over the overall structure. I have followed his advice for the new structure proposed.